src/EnergyPlus/OutdoorAirUnit.hh

```diff
@@ -124,6 +125,7 @@ namespace OutdoorAirUnit {
         std::string ComponentType;
         int ComponentType_Num; // Parameterized Component Types this module can address
         int ComponentIndex;    // Which one in list -- updated by routines called from here
+        UnitarySystems::UnitarySys *compPointer;
```

Comment:
@Myoldmopar what is wrong with this declaration? line 128 OutdoorAirUnit.hh. There is no #include <UnitarySystem.hh> in the header file and Windows MSVS seems OK with that.

Reply:
@jasondegraw do you have time to help me understand the nuance of pointer declaration? Warnings point to line 128 of OutdoorAirUnit.hh

If I add this to the OAEquipList struct do I need to add #include <UnitarySystem.hh> to the OutdoorAirUnit header file? I included that in OutdoorAirUnit.cc but don't know if I need it in both places. This compiles fine on my machines.

UnitarySystems::UnitarySys *compPointer;

Reply:
@rraustad I can try. I'm on my phone right now so it's a bit hard to see what's going on, but once I get back to my desk I can take a look. Are you seeing local warnings or CI warnings?

Reply:
The CI pointed to this line. It seems to run fine and I was expecting a lot of green on the CI. Instead I get a lot of red. I'd like to GoTo meet if possible.